SimpleWAWS.Console/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System;
```

Comment:
it has to be. You can never assume state with tryappservice. All the state is kept in Azure and everytime we read a resoruceGroup into the freelist we need to make sure it's in the desired state.